src/commands/stac-github-import/stac.github.import.ts

```diff
@@ -41,21 +49,34 @@ export const commandStacGithubImport = command({
       defaultValue: () => 'linz/imagery',
       defaultValueIsSerializable: true,
     }),
+    copyOption: option({
```

Comment:
since these directly relate to the copy.ts input parameters could they be linked and validated.

Reply:
It isn't straightforward to link to the copy command because of how the command options are structured, so we have added some validation to the `stac github-import` command.